tpp-private/profileUpload.php

```diff
@@ -0,0 +1,79 @@
+
+    <?php
+
```

Comment:
less space between the opening php tag and the code. Other then that good cooding.